hw13/hw13task1task2/Main_Printable_hw13.java

```diff
@@ -0,0 +1,33 @@
+package hw13task1task2;
+
+
+import static hw13task1task2.Printable_hw13.printableInterfice;
+
+public class Main_Printable_hw13 {
+    public static void main(String[] args) {
+        Printable_hw13[] print = new Printable_hw13[4];
+        print[0] = new Book_hw13("Harry Potter ", "J.K. Rowling");
+        print[1] = new Magazine_hw13("Magazine 1", "17.01.24");
+        print[2] = new Book_hw13("It ", "S. King");
+        print[3] = new Magazine_hw13("Magazine 2", "10.01.24");
+
+        for (Printable_hw13 printable : print) {
+            printUsingMethodReference(printable);
```

Comment:
Method reference is looks like  Printable_hw13::print
Try to replace for each using stream API and method reference